int64 moter1
int64 moter2
int64 moter3
int64 moter4
int64 moter5
